enterprise/frontend/src/embedding-sdk/lib/log-utils.ts

```diff
@@ -0,0 +1,20 @@
+// Note: this functions need to return an array of two elements, when styling
```

Comment:
We started sharing these styles in the logs so I created these helpers, i didn't create a `logWarning` function that called console.warning, because i wanted to keep the correct line showing up in the browser